docs/reference/api/client.txt

```diff
@@ -0,0 +1,142 @@
+.. _client:
+
+################################
+ determined.experimental.client
+################################
+
+..
+   automodule: determined.experimental.client
+
+***********
+ ``Login``
+***********
+
+.. automodule:: determined.experimental.client
+   :members: login
+   :member-order: bysource
+
+***********************
+ ``Create Experiment``
+***********************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: create_experiment
+   :member-order: bysource
+
+********************
+ ``Get Experiment``
+********************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_experiment
+   :member-order: bysource
+
+***************
+ ``Get Trial``
+***************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_trial
+   :member-order: bysource
+
+********************
+ ``Get Checkpoint``
+********************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_checkpoint
+   :member-order: bysource
+
+******************
+ ``Create Model``
+******************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: create_model
+   :member-order: bysource
+
+***************
+ ``Get Model``
+***************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_model
+   :member-order: bysource
+
+****************
+ ``Get Models``
+****************
+
+.. automodule:: determined.experimental.client
+   :noindex:
+   :members: get_models
+   :member-order: bysource
+
+****************
+ ``Checkpoint``
+****************
+
+.. autoclass:: determined.experimental.Checkpoint
```

Comment:
It looks like if we run change each of these lines to:

```rest
.. autoclass:: determined.experimental.client.Checkpoint
```

sphinx will still find the objects, and it will display their names as `determined.experimental.client.Checkpoint`, which seems good